css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,51 @@
+
+.row {
+  display: flex;
+}
+
+.background {
+  width: 80%;
+  margin: auto;
+}
+
+.heading {
+  justify-content: center;
+  font-family: 'DM Sans', sans-serif;
+}
+
+.one-fifth {
+  flex-basis: 20%;
+  flex-wrap: wrap;
+  font-family: 'DM Sans', sans-serif;
+}
+
+.one-fifth > img {
+  width: 95%;
+}
+
+.four > img {
+  width: 106%;
+}
```

Comment:
Instead of doing this individual adjustment of `width`, I would use either `background-image` or `object-fit` for these images, so they can all be the same size.

https://developer.mozilla.org/en-US/docs/Web/CSS/background-image

https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit